docs/userguide/transfer_learning.md

```diff
@@ -1,4 +1,51 @@
 # Transfer Learning
 
-This page will soon contain information about transfer learning.
-In the meantime, please see the [examples](../../examples/examples) instead.
\ No newline at end of file
+BayBE offers the possibility to mix data from multiple campaigns in order to accelerate
+optimization. 
+Using data from multiple campaigns is currently supported when using a Gaussian Process
+Surrogate model and is implemented by the [`TaskParameter`](baybe.parameters.categorical.TaskParameter).
+
+```{admonition} Terminology
+:class: note
+The term "Transfer Learning" is used in a lot of different ways.
+Within BayBE, "Transfer Learning" refers to combining data from multiple campaigns.
+Depending on the field, this might also be known as "contextual learning".
+Further note that the act of combining the data of several contexts can be done via
+multiple models, shared architectures, special kernels in a single model, etc.
+We do not necessarily want to limit to any of these methods, even though BayBE currently
+offers only a single one.
+```
+
+## The role of `TaskParameter`
+
+The `TaskParameter` is used to "mark" the context of an individual experiment. The
+set of all possible contexts is provided upon the initialization of a `TaskParameter`
+by providing them as `values`.
```

Comment:
Needs to have something explaining why youre talking about months below, eg outline the example you chose

Reply:
I have now just replaced that by `Reactor`, is this fine or should this also get some more information?

Reply:
yep